tests/xml.js

```diff
@@ -150,6 +152,18 @@ define([
             eq('<output attr=value />', '<output attr="value" />', false);
         });
 
+        it("should fail on escape unquoted attribute value with <", function () {
+            chai.expect(function () {
+                eq('<output value=2<3 />', '<output value="2&lt;3" />', false);
```

Comment:
I don't know if that input is even valid XML for any parser, but it seems like it might be possible to interpret the first string as the second. However, it will not work with our `normalize` (or `humanize` either) because `fixGTBug` can't handle that input. I added these known edge case failures as documentation of things we know will not work.